examples/2d/elixir_euler_unstructured_quad_ec.jl

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
We need to get better file names: `examples/2d/elixir_euler_ec_curved.jl` vs. `examples/2d/elixir_euler_unstructured_quad_ec.jl` doesn't look good to me (different order of arguments). What happened to you, @sloede? No Mr. Consistency today :sunglasses: 
Could you please add this to some TODO list?

Reply:
I gave up on the names, since it became clear that we need to rename the mesh types anyways, and since we have on the agenda a point on reorganizing the filenames. Once that happens, the mesh type will drop out of the elixir names anyways...